float64[] trajectory
float64[] attitude
